extension/agenthealth/factory.go

```diff
@@ -28,9 +26,7 @@ func NewFactory() extension.Factory {
 func createDefaultConfig() component.Config {
 	return &Config{
 		IsUsageDataEnabled: true,
-		Stats: agent.StatsConfig{
-			Operations: []string{agent.AllowAllOperations},
-		},
+		Stats:              nil,
```

Comment:
Do we need to include this?
```suggestion
		IsStatusCodeEnabled: false,
		Stats:              nil,
```